content/id/docs/concepts/containers/images.md

```diff
@@ -0,0 +1,370 @@
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: private-image-test-1
+spec:
+  containers:
+    - name: uses-private-image
+      image: $PRIVATE_IMAGE_NAME
+      imagePullPolicy: Always
+      command: [ "echo", "SUCCESS" ]
+EOF
+```
+```
+pod/private-image-test-1 created
+```
+
+Jika semuanya berjalan dengan baik, maka setelah beberapa lama, kamu dapat menjalankan:
+
+```shell
+kubectl logs private-image-test-1
+```
+dan lihat pada keluaran perintah:
+```
+SUCCESS
+```
+
+Jika kamu mencurigai ada perintah yang gagal, kamu dapat menjalankan:
+```shell
+kubectl describe pods/private-image-test-1 | grep 'Failed'
+```
+Pada kasus gagal, keluarannya mirip seperti:
+```
+  Fri, 26 Jun 2015 15:36:13 -0700    Fri, 26 Jun 2015 15:39:13 -0700    19    {kubelet node-i2hq}    spec.containers{uses-private-image}    failed        Failed to pull image "user/privaterepo:v1": Error: image user/privaterepo:v1 not found
+```
+
+
+Kamu harus memastikan semua Node di dalam klaster memiliki `.docker/config.json` yang sama. Jika tidak, Pod-Pod
+akan jalan pada beberapa Node saja dan gagal di Node lainnya. Contohnya, jika kamu menggunakan Node _autoscaling_, maka
+setiap templat _instance_ perlu untuk mempunyai `.docker/config.json` atau _mount_ sebuah penyimpanan yang berisi berkas tersebut.
+
+Semua Pod memiliki akses baca (_read_) untuk _image-image_ di registri privat manapun ketika
+_key_ registri privat ditambahkan pada `.docker/config.json`.
+
+### _Image_ Pra-unduh 
+
+{{< note >}}
+Jika kamu jalan di Google Kubernetes Engine, maka akan ada `.dockercfg` pada setiap Node dengan kredensial untuk Google Container Registry. Kamu dapat menggunakan cara ini.
+{{< /note >}}
+
+{{< note >}}
+Cara ini cocok jika kamu dapat mengontrol konfigurasi Node. Cara ini tidak akan
+bisa berjalan dengan baik pada GCE, dan penyedia cloud lainnya yang tidak menggantikan
+Node secara otomatis.
+{{< /note >}}
+
+Secara _default_, kubelet akan mencoba untuk mengunduh setiap _image_ dari registri yang dispesifikasikan.
+Hanya saja, jika properti `imagePullPolicy` diatur menjadi `IfNotPresent` atau `Never`, maka
+sebuah _image_ lokal digunakan.
+
+Jika kamu ingin memanfaatkan _image_ pra-unduh sebagai pengganti untuk otentikasi registri,
+kamu harus memastikan semua Node di dalam klaster memiliki _image_ pra-unduh yang sama.
+
+Cara ini bisa digunakan untuk memuat _image_ tertentu untuk kecepatan atau sebagai alternatif untuk otentikasi untuk sebuah registri privat.
+
+Semua Pod akan mendapatkan akses baca ke _image_ pra-unduh manapun.
+
+### Tentukan ImagePullSecrets pada sebuah Pod
+
+{{< note >}}
+Cara ini merupakan cara yang direkomendasikan saat ini untuk Google Kubernetes Engine, GCE, dan penyedia cloud lainnya yang
+secara otomatis dapat membuat Node.
+{{< /note >}}
+
+Kubernetes mendukung penentuan _key_ registri pada sebuah Pod.
+
+#### Membuat sebuah Secret dengan Docker Config
+
+Jalankan perintah berikut, ganti dengan nilai huruf besar yang tepat:
```

Comment:
```suggestion
Jalankan perintah berikut, ganti nilai huruf besar dengan yang tepat:
```

Reply:
fixed in https://github.com/kubernetes/website/pull/19784/commits/3fb85d866101c08ce75ea0265beac4929e73f29c